src/components/ProductDetails/RelatedProductsSections.jsx

```diff
@@ -15,14 +15,20 @@ const RelatedProductsSections = () => {
   };
 
   return (
+    <Row justify="center" align="middle" >
+    <Col align="middle" justify="center">
+      <Typography.Title style={{ marginBottom: 20 }} level={3}>RELATED PRODUCTS</Typography.Title>
+      <Row gutter={[16, 16]} >
+        {RelatedProductsList.map((product, index) => (
+          <Col   key={index}>
```

Comment:
don't use index as key
use product.id or something like that

Reply:
Will change it